### PR TITLE
timeout in local tests, ctrl+c correctly stop action.

### PR DESCRIPTION
**Pull request by romanprog**

### CODE REVIEW

File: tests/timeout.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+sleep 1
+ACTION_TIMEOUT=$1
+CONTAINER_NAME=$2
+
+  for seconds in $(seq 1 ${ACTION_TIMEOUT}); do
+    if [ -z  "$(docker ps| grep ${CONTAINER_NAME})" ]; then
```
Comment (MaxymVlasov) [inline tests/timeout.sh]: https://github.com/koalaman/shellcheck/wiki/SC2143

Also, looks like you not use `shellcheck`, `.editorconfig` and `pre-commit` how it recommended by our [style guide](https://github.com/shalb/cluster.dev/blob/master/docs/style-guide.md)

`shellcheck` usage via CLI is [simple](https://github.com/koalaman/shellcheck#from-your-terminal).
Feel free ask me if you need any help with it.

Also, our indent for shell script was [set to 4](https://github.com/shalb/cluster.dev/blob/fea06e6d3c0aa49324be8c4cad7fc890988dd7dd/.editorconfig#L11), but your editor set it to 2.

P.S. Some recommendations from `shellcheck`:

File: tests/config.example.sh [hunk 2]
```diff
@@ -2,4 +2,6 @@
 
 AWS_ACCESS_KEY_ID=""
 AWS_SECRET_ACCESS_KEY=""
-GH_ACTION_WORKFLOW_PATH="./.cluster.dev/gh-5.yaml" # Relative path from project root
+# Timeout in seconds.
+ACTION_TIMEOUT="600"
```
Comment (MaxymVlasov) [inline tests/config.example.sh]: I recommend add more info for what propose this var needed
Like this
```bash
GH_ACTION_RUN_TIMEOUT="600"
```
or this
```bash
ACTION_TIMEOUT="600" # After timeout Github Action will be stopped
```

Comment (voatsap) [reply tests/config.example.sh]: Agree with that proposal

File: tests/timeout.sh (hunk 1 above)
Comment (romanprog) [reply tests/timeout.sh]: I can not use pre-commit. Problem with installing in my system. 
All proposed by shellcheck fixes are not relevant. Besides the syntax error in the word. But fixed.

File: tests/tests.sh [hunk 3]
```diff
@@ -17,8 +18,27 @@ docker build -t ${DOCKER_IMAGE_NAME} .
 readonly USER="${AWS_ACCESS_KEY_ID}"
 readonly PASS="${AWS_SECRET_ACCESS_KEY}"
 readonly WORKFLOW_PATH="${GH_ACTION_WORKFLOW_PATH}"
+readonly TIMEOUT="${ACTION_TIMEOUT}"
+
+# Trap ctrl+c to remove docker container and kill timeout script.
+trap ctrl_c INT
+function ctrl_c() {
```
Comment (MaxymVlasov) [inline tests/tests.sh]:
```suggestion
function ctrl_c {
```

Comment (MaxymVlasov) [reply tests/tests.sh]: >The function keyword is extraneous when “()” is present after the function name, but enhances quick identification of functions.

https://google.github.io/styleguide/shellguide.html#function-names

As I understand, we should use or `function name {}` or `name() {}`

In all other funtions used `function name {}` style

Comment (romanprog) [reply tests/tests.sh]: done

**Pull request merged**